.gitmodules

```diff
@@ -7,3 +7,6 @@
 [submodule "externals/rtaudio"]
 	path = externals/rtaudio
 	url = https://github.com/thestk/rtaudio.git
+[submodule "externals/oscpp"]
```

Comment:
I consider the use of git submodules to be deprecated, in favor of using meson's subprojects or (in the case of Simple-FFT which is a also header only) just checking things directly into the repo's "externals" directory. People working down-stream complained when we first added Simple-FFT as a git submodule. So I think this should also be added not as a subproject but just by checking the header files in directly.

Reply:
I think it would also be ok if this was wrapped as a meson subproject, but then it should probably either reference the source repo or be a fork in the jacktrip project, if necessary.

Reply:
I added the source code of oscpp into `externals/oscpp` directly + removed the submodule declaration